src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -191,24 +205,24 @@ class DiskImageEditor extends Component {
   }
 
   changeAlias ({ target: { value } }) {
-    this.setState({ alias: value })
+    this.setState((state) => ({ values: { ...state.values, alias: value }, errors: { ...state.errors, 'alias': '' } }))
```

Comment:
This would allow for immediate feedback as the user types instead of waiting for them to click the ok button:
```suggestion
    this.setState((state) => ({ 
      values: { ...state.values, alias: value }, 
      errors: { ...state.errors, 'alias': isDiskNameValid(value) ? '' : msg.pleaseEnterValidDiskName() },
    }))
```